internal/host/transition_test.go

```diff
@@ -75,10 +75,11 @@ func createValidatorCfg() *hardware.ValidatorCfg {
 		},
 		VersionedRequirements: hardware.VersionedRequirementsDecoder{
 			"default": {
-				Version:            "default",
-				MasterRequirements: &defaultMasterRequirements,
-				WorkerRequirements: &defaultWorkerRequirements,
-				SNORequirements:    &defaultSnoRequirements,
+				Version:                "default",
+				MasterRequirements:     &defaultMasterRequirements,
+				WorkerRequirements:     &defaultWorkerRequirements,
+				SNORequirements:        &defaultSnoRequirements,
+				EdgeWorkerRequirements: &defaultWorkerRequirements,
```

Comment:
Is it a copy paste issue or it should be defaultWorkerRequirements?

Reply:
in this case it just doesn't matter. If not exists it should show worker reqs. So it just doesn't matter in this specific case and was pity to create some new unused variable